packages/docs/components/layout/mdx-provider-components.js

```diff
@@ -72,10 +86,10 @@ export const H3 = addLinkAnchor(({ className, ...props }) => (
 export const H4 = addLinkAnchor(({ className, ...props }) => (
   <Title className={classnames(className, 'pf-u-mb-md pf-u-mt-md')} headingLevel="h4" {...props} />
 ));
-const Table = (props) => {
+export const Table = (props) => {
   const classes = useTableStyles();
   return (
-    <Card className={classnames('pf-u-mb-lg', classes.card)}>
+    <Card className={(classnames('pf-u-mb-lg'), classes.card)}>
```

Comment:
Was there any specific reason for this change? I am not sure about the wrapping parentheses. I feel like it will remove the second class from the class list.

Reply:
already fixed